_posts/2017-04-05-airpods-bluetooth-interference-mac.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title:  "Troubleshoot: Poor AirPod Connection with Mac Mini"
+date:   2017-04-05 09:22:07 -0400
+categories: jekyll tech ccannon Apple AirPods MacMini
+---
+I recently received a pair of AirPods as a birthday present and I must say, they are incredible. However, I experienced some difficulty right away with connecting them to my Mac Mini. They would pair, but sound would come over broken, jumping from one ear to the other, with nothing clear coming through. I decided to contact Apple Support after trying the most basic troubleshooting I could think of: unpairing and repairing the AirPods from my AppleID, and pairing the AirPods directly to the Mac Mini.
```

Comment:
The AirPods would pair to my Mac Mini, but the sound quality was less than desirable. I experienced poor connection quality, with a lack of clarity and sound jumping from one ear to the other. It seemed strange considering the AirPods seemed to connect fine with my other Apple products, such as my phone, Apple Watch, MacBook, and iPad (You could also just say other Apple devices so people don't know how loaded you are with Apple devices lol)
After trying to basic troubleshooting such as unpairing and repairing from my AppleID, pairing directly with the Mac Mini I decided it was time to get some more(or professional) help from Apple Support.
I think it flows better this way, but I mean its your blog haha